spec/system/report_request_for_comment_comments_system_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Report RfC comments for inappropriate content', :js do
+  let(:user) { create(:learner) }
+
+  before do
+    stub_const('CommentPolicy::REPORT_RECEIVER_CONFIGURED', reports_enabled)
+    visit(sign_in_path)
+    fill_in('email', with: user.email)
+    fill_in('password', with: attributes_for(:learner)[:password])
+    click_button(I18n.t('sessions.new.link'))
+    visit(request_for_comment_path(create(:rfc_with_comment)))
+    find('span.ace_icon').click
+  end
+
+  context 'when reporting is enabled' do
+    let(:reports_enabled) { true }
+
+    it 'allows reporting of RfCs' do
```

Comment:
This seems to be a copy-paste leftover. Could you please check the whole file for those?
```suggestion
    it 'allows reporting the comment' do
```

Reply:
Sorry!